js/loadTrack.js

```diff
@@ -0,0 +1,24 @@
+$('.trackLink').click(function(e) {
+    var link = e.target.dataset.track
+        
+    if  (link === '1' && $('#track1').length === 0) {
+        $('#container').append($('<div>').load('includes/trackLayout1.html'))
+    } else if (link === '1' && $('#track1').length !== 0) {
```

Comment:
else if but no else at the end?

Reply:
omitted else if and replaced it with else and abstracted into function